cli/spec/support/output_helpers.rb

```diff
@@ -55,16 +55,7 @@ module OutputHelpers
         @real.zip(@expected) do |real, expected|
           line += 1
           fields = real.split(/\s{2,}/)
-          with_regex = expected.each_index.select { |i| expected[i].kind_of?(Regexp) }
-          without_regex = expected.each_index.reject { |i| with_regex.include?(i) }
-          if with_regex.empty?
-            regexes_pass = true
-          else
-            regexes_pass = without_regex.all? do |i|
```

Comment:
Oops... this should probably have been `with_regexp`? Seems like this wasn't actually checking the regexp matches, which is why the specs were passing in travis even with the wrong `--long` format.

Reply:
probably